composer.json

```diff
@@ -39,9 +39,9 @@
     "require": {
     },
     "require-dev": {
-        "cakephp/cakephp": "~3.0",
+        "cakephp/cakephp": "3.4.*",
```

Comment:
Keep this unchanged, highest available cakephp release supported on the platform will be automatically used.